### PR TITLE
Durak

### PR DESCRIPTION
**Pull request by glmcz**
Zadost o PR prosim prosim prosim ...

### CODE REVIEW

**Comment by glmcz** [comment]
@himdel  @romanblanco  @ZitaNemeckova  @PanSpagetka  @lpichler

File: glmcz.md [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
 
 
-Your favorite fruit: ...
+Please change me Laska je laska 
 
-Your favorite city: ...
 
-Your favorite vegetable: ...
+
+Your favorite fruit: apple
```
Comment (lpichler) [inline glmcz.md:9]: 👍

File: glmcz.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+
+Your favorite city: Znojmo
+
+Your favorite vegetable: carrot(mrkev)
 
 Your favorite color: ...
 
-Your favorite icecream: ...
+Your favorite icecream: no one (zadna nemam rad zmrzlinu, lepsi je si koupit tvaroch a dat si do toho ovoce...)
```
Comment (lpichler) [inline glmcz.md:17]: 🤔

**Comment by himdel** [comment]
- [x] git rebase master - solving conflicts **3pts**
- [x] git rebase -i master (squash) **3pts**
- [x] git rebase -i master (reword) **2pts**
- [x] git rebase -i master (drop) **1pts**
- [x] git rebase -i master (edit) **2pts**
- [x] **one** of instructor mentioned **1.5pts**

= 12.5 points

(But feel free to update the PR, you can still get the points.)

**Comment by glmcz** [comment]
@himdel  git rebase -i master (edit) 2pts updated PR please check it

**Comment by ZitaNemeckova** [comment]
@glmcz Good work :) Points added.

**Pull request closed without merging**